infra/Setup-SplitExperimentationEntraApp.ps1

```diff
@@ -44,6 +44,8 @@ function Setup-SplitExperimentationEntraApp
         $app = Get-SplitApp
     }
 
+    az ad app owner add --id $app.id --owner-object-id $userObjectId
```

Comment:
Need to confirm this is idempotent. Should be able to run this script over and over.

Reply:
Discussed offline, will remove this until the cause is better understood. Right now this looks to be adding ones own self as an owner, even though the caller must be an owner to add an owner.